frontend/src/modules/editor/components/TranslationLength.js

```diff
@@ -0,0 +1,87 @@
+/* @flow */
+
+import * as React from 'react';
+
+import './TranslationLength.css';
+
+import type { DbEntity } from 'modules/entitieslist';
+
+
+type Props = {|
+    entity: ?DbEntity,
+    translation: string,
+|};
+
+
+/*
+ * Shows translation length vs. original string length, or countdown.
+ * 
+ * Countdown is current only supported for LANG strings, which use special
+ * syntax in the comment to define maximum translation length. MAX_LENGTH
+ * is provided for strings without HTML tags, so they need to be stripped.
+ */
+export default class TranslationLengthBase extends React.Component<Props> {
+    getLimit() {
+        const entity = this.props.entity;
+
+        if (!entity) {
```

Comment:
If you make this `if (!entity || entity.format !== 'lang') {` you can simplify the logic and avoid doing the `split` when the format is not LANG.